src/docs/Icons/Icons.tsx

```diff
@@ -10,10 +10,10 @@ import {
 import clipboardCopy from 'clipboard-copy';
 import { suomifiDesignTokens } from 'suomifi-design-tokens';
 
-const baseIconKeys = baseIcons as string[];
-const illustrativeIconKeys = illustrativeIcons as string[];
-const doctypeIconKeys = doctypeIcons as string[];
-const logoIconKeys = logoIcons as string[];
```

Comment:
`as string[]` is no longer needed since the correct typing is coming from the icons library already